_applications/ceph-source-example.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Ceph Source Example
+link: ceph-source-example
+weight: 100
+labels:
+- Python
+- Ceph
+- S3
+- Jupyter
+layout: application
+menu_template: menu_tutorial_application.html
+menu_items: lightning
+description: |-
+  This is an example of how to connect your application to data in Ceph using S3 API.
+project_links:
+- https://github.com/rimolive/ceph-spark-integration
+---
+<h1 id="introduction">Introduction</h1>
+
+Processing data stored in an external object store is a practical and
+popular way for an intelligent application to operate.
+
+This is an example of the key pieces needed to connect your
+application to data in Ceph using S3 API. It is presented as steps in a Jupyter
+notebook. To run this example, you need ```vagrant``` and ```ansible```.
```

Comment:
imo, we should keep these examples as simple as possible leveraging the "Get Started" page instructions. i think we should stay away from creating alternate install methodologies.

are vagrant and ansible for installing openshift and ceph?